test-apps/display-test-app/src/backend/Backend.ts

```diff
@@ -53,7 +53,7 @@ class DisplayTestAppRpc extends DtaRpcInterface {
 
   public override async writeExternalSavedViews(bimFileName: string, namedViews: string): Promise<void> {
     if (ProcessDetector.isMobileAppBackend && process.env.DOCS) {
-      const docPath = process.env.DOCS;
+      const docPath = process.env.DOCS;  // What is DOCS?

```

Comment:
@markschlosseratbentley @pmconne I didn't dig too deep but which is this used for? Do we still need it?

Reply:
This is the Documents path on an iPad or iPhone for the specific app running.  This is the writeable directory dta will have access to.  @khanaffan added this variable, I believe... maybe he has some suggestions.

Reply:
FYI, in order for display-test-app to save external saved views, this is needed.  I am not sure if we will need some other way to get the Documents path than an environment variable due to the changes in this PR.

Reply:
The variable should still work with this PR just won't be accessible on the frontend.

Why do you need this one to save external saved views?

Reply:
On iOS, we need a directory to which we have permission to write.

Reply:
Oh okay, thought it was to save the views to a service not just locally on the disk. Sounds good.

Do you want me to switch it to `IMJS_DOCS` or leave it as is? It'll technically work either way.

Reply:
Safest to leave it I think. We only need it on the backend anyway.

Reply:
Sounds good. Resolving...

Reply:
Can you revert your incredulous "Why???" and "What is DOCS?" comments?

Reply:
Updated to a more useful comment for the DOCS and removed the Why